mitmproxy/contentviews/image.py

```diff
@@ -19,6 +21,10 @@ class ViewImage(base.View):
     ]
 
     def __call__(self, data, **metadata):
+        if imghdr.what(io.BytesIO(data)) == 'png':
+            img = metadata.Metadata.open(io.BytesIO(data))
```

Comment:
I feel your pain - the kaitai API documentation is rather shitty on that end. From [here](https://github.com/kaitai-io/kaitai_struct_python_runtime/blob/2a559b66d75d102a19a72a5c6544c850b39dd6f6/kaitaistruct.py#L25):

```
Png(KaitaiStream(data))
```